Description: Change documentation theme to one not requiring PHP and Wordpress
Author: Harlan Lieberman-Berg <H.LiebermanBerg@gmail.com>
Forwarded: not-needed

--- a/docsite/conf.py
+++ b/docsite/conf.py
@@ -101,7 +101,7 @@
 # -----------------------
 
 html_theme_path = ['_themes']
-html_theme = 'aworks'
+html_theme = 'bootstrap'
 html_short_title = 'Ansible Documentation'
 
 # The style sheet to use for HTML and HTML Help pages. A file of that name
